package.json

```diff
@@ -5,8 +5,12 @@
   "private": true,
   "scripts": {
     "dev": "NODE_ENV=development DEBUG=metalsmith-timer,metalsmith-algolia,metalsmith-layouts,metalsmith-assets node index.js",
-    "build": "NODE_ENV=production DEBUG=metalsmith-timer,metalsmith-algolia node index.js",
-    "webpack": "webpack --progress --colors"
+    "build": "DEBUG=metalsmith-timer,metalsmith-algolia node index.js",
```

Comment:
okay so the preexistence of `NODE_ENV=production` concerns me that metalsmith is already relying on this value for other things under the hood. We may want to create a new env var like `DOCS_ENV` and rely on that

Reply:
As far as I can see metalsmith is doing nothing with it (https://github.com/segmentio/metalsmith/search?q=NODE_ENV)
But we should do it anyway as probably some other tools in our toolchain will rely on that or will do in the future.